addon/templates/components/power-select/multiple.hbs

```diff
@@ -12,7 +12,9 @@
       removeOption=(action "removeOption" dropdown)
       handleKeydown=(action "handleKeydown" dropdown)
     )) as |select|}}
-
+      {{#if createEnabled}}
+        <ul class="ember-power-select-options "><li {{action "create" dropdown searchText}} class="ember-power-select-option">Add {{searchText}}...</li></ul>
+      {{/if}}
```

Comment:
Please make the message a computed property, like the other messages. One might want to customize this (e.g non-english users).